### PR TITLE
Compute tumor volume from binary mask and list of centroids

### PR DESCRIPTION
**Pull request by vessemer**
## Description
This method computes the volume of connected components placed at the specified centroids, via connected component analysis using `scipy.ndimage.label`.

## Reference to official issue
Calculate tumor volume from pixel masks [#13](https://github.com/concept-to-clinic/concept-to-clinic/issues/13) 

## How Has This Been Tested?
The unit test for `predict` has been added  to test for the volume calculation over the different mask cases.

## CLA
- [x] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

File: prediction/src/algorithms/segment/trained_model.py [hunk 1]
```diff
@@ -2,30 +2,29 @@
 """
     algorithms.segment.trained_model
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
```
Comment (lamby) [inline prediction/src/algorithms/segment/trained_model.py]: Please try and avoid making whitespace changes that are unrelated to the commit message contents. Feel free to make them, but separating them out into their own commit makes it much easier to review stuff!

File: prediction/src/tests/test_trumor_volume.py [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    assert volumes_calculated == [100, 20, 30]    
+    shutil.rmtree(os.path.dirname(path))
+    
+
+
+def test_calculate_volume_on_overlapped_connected_components():
+    centroids = [[0, 0, 0], [0, 0, 0], [45, 45, 12]]
+    centroids = [{'x': centroid[0], 'y': centroid[1], 'z': centroid[2]} for centroid in centroids]
+    mask = generate_mask(shape=[50, 50, 29], centroids=centroids, volumes=[100, 20, 30])
+#   The balls area must be 100 + 30, since first ball have overlapped with the second one
```
Comment (lamby) [inline prediction/src/tests/test_trumor_volume.py]: Should this not be indented? :)

File: prediction/src/algorithms/segment/trained_model.py (hunk 1 above)
Comment (vessemer) [reply prediction/src/algorithms/segment/trained_model.py]: Thanks for noted that. I will stick with it further.

**Comment by vessemer** [comment]
The Travis CI build failed due to LFS issues [#69](https://github.com/concept-to-clinic/concept-to-clinic/issues/69).

**Comment by vessemer** [comment]
I've refreshed the pull request [#78](https://github.com/concept-to-clinic/concept-to-clinic/pull/78) and made it clearer.

**Pull request closed without merging**